tarpc/Cargo.toml

```diff
@@ -27,43 +27,42 @@ travis-ci = { repository = "google/tarpc" }
 
 [dependencies]
 fnv = "1.0"
-futures-preview = { version = "0.3.0-alpha.18" }
+futures = "0.3"
 humantime = "1.0"
 log = "0.4"
 pin-project = "0.4"
 raii-counter = "0.2"
 rand = "0.7"
-tokio-timer = "0.3.0-alpha"
 serde = { optional = true, version = "1.0", features = ["derive"] }
-tokio = { optional = true, version = "0.2.0-alpha" }
+tokio = { optional = true, version = "0.2", features = ["time"] }
+tokio-util = { optional = true, version = "0.2" }
 tarpc-plugins = { path = "../plugins" }
 
 async-bincode = { optional = true, version = "0.4" }
 futures-legacy = { optional = true, version = "0.1", package = "futures" }
-futures-test-preview = { optional = true, version = "0.3.0-alpha" }
+futures-test = { optional = true, version = "0.3" }
 tokio-io = { optional = true, version = "0.1" }
 tokio-tcp = { optional = true, version = "0.1" }
 
-tokio-net = { optional = true, version = "0.2.0-alpha" }
-tokio-serde-json = { optional = true, version = "0.3" }
+tokio-serde = { optional = true, version = "0.5", features = ["json"] }
```

Comment:
Does it need `features = [json]` if the json feature is enabled by other the json-transport feature?